sdk/provisioning/Azure.Provisioning.AppConfiguration/src/Azure.Provisioning.AppConfiguration.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains the core functionality for defining Azure infrastructure with dotnet code.</Description>
+    <Version>0.1.0-beta.1</Version>
```

Comment:
Would it make our lives easier in the short term to declare a constant for these values in `dir.props`?

Reply:
Actually, I don't think we will ship the individual libraries lockstep and they have already diverged from the main package.